corehq/motech/repeaters/expression/repeaters.py

```diff
@@ -246,11 +246,23 @@ def _error_response(error_json):
         'An error occurred.'
 
         """
+        # The ArcGIS REST API documentation does not give the error
+        # response schema, so we have to guess based on what we've seen.
+
+        # `status_code` is required for us to make decisions about the
+        # repeat record.  If `error_json` does not include "code", then
+        # use 500 so that the repeat record will be sent again later.
+        status_code = error_json.get('code', 500)
+
+        # `reason` is what is shown in the Repeat Records Report under
+        # the "Responses" button. If `error_json` is missing "message",
+        # then set a value that is more useful to users than no message.
```

Comment:
Nice comments.

> If `error_json` is missing "message", then set a value that is more useful to users than no message.

How about setting it to something that is true like "Missing error message" or "No error message"?

Reply:
haha, may be that just came from personal issues I have reading too much into words. "Unspecified" could mean different for me.
I just thought using a message that users see more often could be useful and hence the suggestion.